products/green_fast_track/models/intermediate/spatial/shadows/int_spatial__shadow_hist_resources.sql

```diff
@@ -1,13 +1,12 @@
-WITH all_natural_resources AS (
+WITH all_shadow_resources AS (
 {{ dbt_utils.union_relations(
     relations=[
-        ref('stg__lpc_landmarks'),
-        ref('stg__nysshpo_historic_buildings'),
+        ref('int_spatial__historic_resources'),
```

Comment:
This is I think why I'm a little less crazy about the name of `int_spatial__historic_resources` - lpc_scenic_landmarks seems like a historic resource to me. 

Maybe `historic_lots`? `historic_landmarks_and_buildings`? Just something a little more specific would be helpful, until we're at this dataset, which really is all historic resources.

Lemme know if you disagree obvs, I'm coming in with less context

Reply:
(per chat in-person) EARD/Planning Support named the questions and decided which sources go into each question, unfortunately Shadow Historic Resources _isn't_ just "buffer Historic Resources"

separate from these changes, we have a discrete task to review alignment of the build with the intentions of EARD/Planning Support. if it turns out that Shadow Historic Resources is just "buffer Historic Resources", this idea makes more sense

that review should center on our [`variables.csv`](https://github.com/NYCPlanning/data-engineering/blob/main/products/green_fast_track/seeds/variables.csv) because the [Source Data excel file](https://nyco365.sharepoint.com/:x:/r/sites/NYCPLANNING/itd/edm/_layouts/15/Doc.aspx?sourcedoc=%7BBAE15377-87A6-4188-9637-36A89AC1DF98%7D&file=CEQR_Type_II_Data_Source_Review.xlsx&action=default&mobileredirect=true) isn't really as clear about "source data in each question"